team-manager/pom.xml

```diff
@@ -9,8 +9,36 @@
         <artifactId>knudev</artifactId>
         <version>0.0.1-SNAPSHOT</version>
     </parent>
-
     <artifactId>team-manager</artifactId>
 
+    <dependencies>
+        <dependency>
+            <groupId>ua.knu</groupId>
+            <artifactId>knudev-common</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
As already said, you already have trhis in common moduke

Reply:
ok